react/Sidebar/index.jsx

```diff
@@ -2,12 +2,28 @@ import React from 'react'
 import cx from 'classnames'
 import PropTypes from 'prop-types'
 import styles from './styles.styl'
+import { useSetFlagshipUI } from '../hooks/useSetFlagshipUi/useSetFlagshipUI'
+import { useTheme } from '@material-ui/core'
 
-const Sidebar = ({ children, className, ...restProps }) => (
-  <aside className={cx(styles['o-sidebar'], className)} {...restProps}>
-    {children}
-  </aside>
-)
+const Sidebar = ({ children, className, ...restProps }) => {
+  const theme = useTheme()
+
+  useSetFlagshipUI(
```

Comment:
nit: add a test checking useSetFlagshipUI is called with correct arguments, when this component is rendered